### ISSUE TITLE
HTML output not working

### ISSUE THREAD

**Issue by phatmann** [ChordPro/chordpro#104]
**Describe the bug**
When I try to output to an HTML file, I get "No backend for HTML" error.

**To Reproduce**
1. Run a ChordPro command with --output="MySong.html" option

**Expected behavior**
HTML file is created.

**Sample files and screenshots**
```
$ chordpro MySong.cho --output=MySong.html

No backend for HTML
Can't locate App/Music/ChordPro/Output/HTML.pm in @INC
```

```
$ chordpro --version
This is ChordPro version 0.974
```

**Additional context**
When I look in the code used to build ChordPro in the .cpan directory, I see that HTML.pm is not in the Output directory.

I know HTML is experimental, but the release docs say it is included.

I installed ChordPro via cpan.

**Comment by sciurius** [comment]
HTML output is currently only available in the dev branch and will be part of the next release.
If you run ChordPro from git you can try the HTML backend.

**Comment by phatmann** [comment]
Thanks for letting me know. However, the release notes for 0.95 say that HTML support is part of the release. Can we edit those release notes?

**Comment by sciurius** [comment]
I don't see that anywhere in the release notes.
For 0.94 there's a note "Add html to list of recognized output types (even though experimental)". This means that an output file type, e.g. `--output=foo.html` will be understood to be intended for the (not yet existing) HTML backend.

Do you really think it is important to retrofit old release notes? You can try the HTML backend from git, and the next release (that includes HTML) is near.

**Comment by phatmann** [comment]
Updating the release notes is important to avoid the misunderstanding I went through. We could change it to say:

"Add html to list of recognized output types (note: the HTML backend is experimental and not yet supported)”

Or just take that whole line out. Anything to avoid confusion around HTML support.

**Comment by sciurius** [comment]
Fixed in 0.95.
